kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/util/IfcfgConfig.java

```diff
@@ -0,0 +1,169 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2018 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech
+ *     Benjamin Cabé - fix for GH issue #299
+ *******************************************************************************/
+
```

Comment:
Could you please update the copyright (2018 only) and remove Benjamin Cabè from the contributors?